### DISCUSSION TITLE
How do you install the HWinfo plugin

### DISCUSSION THREAD

**Discussion by Kolchak72** [Rem0o/FanControl.Releases#705] [Q&A] [answered]
I'm trying to install the HWinfo plugin so I can set a fan curve to a sensor Fan Control can't detect. What is "Compile the solution."?

**Comment by xhul-dev** [comment, marked as answer]
Hello there.
Maybe that's the answer you're looking for: https://github.com/Rem0o/FanControl.Releases/discussions/631

**Comment by Kolchak72** [comment]
Thanks, any idea why unblocking the dll is not in the instructions?

**Reply by Fr0stX76** [reply]
I created a full guide here https://github.com/Rem0o/FanControl.Releases/discussions/714
